### PR TITLE
Ratification Polls

### PR DESCRIPTION
**Pull request by Gala-G**
Ratification Polls for:
MIP66
MIP67

+ erase discarded poll

### CODE REVIEW

File: governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to Adding Adding Events Core Unit (EVENTS-001) being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP32
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md:28]: "MIP39c2-SP32: Adding Events Core Unit (EVENTS-001)".

We should do the same with the rest, too

File: governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Events Core Unit will handle planning, organization, and execution of MakerDAO branded events. Our vision is to use an exhaustive list of themes, visuals, set dressing, technology and branding to carry out events to the highest standards for the MakerDAO community.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/master/MIP39/MIP39c2-Subproposals/MIP39c2-SP32.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md:35]: This should be a link to the latest commit. We should do the same with the rest, too

File: governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md:2]: "MIP67"

File: governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-11T16:00:00
+---
+# Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md:15]: "MIP67"

File: governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+discussion_link: https://forum.makerdao.com/t/mip67-methodology-and-review-process-for-structured-finance-transactions/13737
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-11T16:00:00
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md:13]: Change end date

File: governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+discussion_link: https://forum.makerdao.com/t/mip39c2-sp32-adding-events-core-unit-events-001/13780
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-11T16:00:00
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md:13]: Change end date

File: governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
+---
+title: Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022
+summary: Coordinate and execute MakerDAO branded events at Crypto/DEV Conference locations.
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md:3]: "These proposals onboard the Events Core Unit (EVENTS-001)..."

File: governance/polls/Ratification Poll for Pairwyse Licensure (MIP66) - April 11, 2022.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+discussion_link: https://forum.makerdao.com/t/mip66-pairwyse-licensure/13554
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-04-11T16:00:00
+end_date: 2022-04-11T16:00:00
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Pairwyse Licensure (MIP66) - April 11, 2022.md:13]: Change end date

File: governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip39c2-sp32-adding-events-core-unit-events-001/13780)
+
+### MIP40c3-SP66: Events Core Unit DAI Budget (EVENTS-001)
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP66 adds the DAI budget for the Core Unit EVENTS-001: Events. This budget will allocate DAI to the CU. The one-off seed transfer will be 149,692 DAI with the monthly transfers being 62,371.50 (12 transfers total). The total CU budget amounts to 898,150 DAI annually.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/f7588070945cab9ae27ead086e1126a4d343171a)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md:46]: This is the link: https://github.com/makerdao/mips/blob/1a75df31c022c3049a2f623ed3178a10457ff5fb/MIP39/MIP39c2-Subproposals/MIP39c2-SP32.md

File: governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+The community may vote in this poll to express support or opposition to Adding Events Core Unit (EVENTS-001) being accepted and implemented in the Maker Protocol.
+
+### MIP39c2-SP32: Adding events Core Unit (EVENTS-001)
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*The Events Core Unit will handle planning, organization, and execution of MakerDAO branded events. Our vision is to use an exhaustive list of themes, visuals, set dressing, technology and branding to carry out events to the highest standards for the MakerDAO community.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/1a75df31c022c3049a2f623ed3178a10457ff5fb)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md:35]: Use this link: https://github.com/makerdao/mips/blob/1a75df31c022c3049a2f623ed3178a10457ff5fb/MIP39/MIP39c2-Subproposals/MIP39c2-SP32.md

File: governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+### MIP41c4-SP33: Events Core Unit Facilitator - Jon Perry (EVENTS-001)
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*This sub proposal onboards Jon Perry as the Facilitator for EVENTS-001. As the facilitator, he will be responsible for ensuring the Events CU achieves its mandate of producing branded events for MakerDAO to the highest level of quality, transparency and professionalism.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/bf522ee865929361524eddee332ef3a54d84bcb0)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP41c4SP33)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Adding Events Core Unit (EVENTS-001) - April 11, 2022.md:58]: Use this link: https://github.com/makerdao/mips/blob/bf522ee865929361524eddee332ef3a54d84bcb0/MIP41/MIP41c4-Subproposals/MIP41c4-SP33.md

File: governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP67 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP67 intends to (1) formally approve the Methodology for Review of MIP6 Structured Finance Transactions 17 published by the Real-World Finance Core Unit, and (2) introduce a high-level process that each MIP6-compliant application should undergo in order to receive, eventually, a positive endorsement from the relevant real-world CU stakeholder groups. E.g. RWF-001, SES-incubated Lending Oversight team, future CUs.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/7ea0b7ab8198bb5ff018a15f57e9766427ace6ca)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Methodology and Review Process for Structured Finance Transactions (MIP 67) - April 11, 2022.md:33]: Use this link: https://github.com/makerdao/mips/blob/7ea0b7ab8198bb5ff018a15f57e9766427ace6ca/MIP67/MIP67.md

File: governance/polls/Ratification Poll for Pairwyse Licensure (MIP66) - April 11, 2022.md [hunk 13]
```diff
@@ -0,0 +1,59 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP66 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*Pairwyse is an active protocol and blueprint for creating bespoke fixed-rate DAI lending and investment Agreements on MakerDAO between borrowers and lenders. We wish to thank the Maker Community for its previous feedback concerning the negative impact of patents on open source projects and potential concentration risks posed by fixed-rate innovations currently incubating in the Maker ecosystem. In posting this MIP, we wish to directly address these shared concerns with a fresh proposal we feel is favorable for MakerDAO, its contributors, its users and all current and future DeFi innovators throughout the collective ecosystem.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/commit/2e55a730e76594880d6f4b69104b0816e7b5720b)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for Pairwyse Licensure (MIP66) - April 11, 2022.md:33]: Use this link: https://github.com/makerdao/mips/blob/2e55a730e76594880d6f4b69104b0816e7b5720b/MIP66/MIP66.md

**Review by prose11 — COMMENTED** [review]
Nice work! Only additional changes are the links mentioned by Pablo, which I will cover in my PR with all the polls. Thank you!

**Comment by Gala-G** [comment]
Ok. I will pay attention next time I do Ratification Polls. Pablo mentioned initially "last commit" and I thought it was the link you get when you go to the "last commit" on the top right of every file on git.

**Pull request merged**